tools/tests.xml

```diff
@@ -5,6 +5,8 @@
   <test name="terminalCore" type="unit" binary="Terminal.Core.Unit.Tests.dll" />
   <test name="terminalApp" type="unit" binary="UnitTests_TerminalApp\Terminal.App.Unit.Tests.dll" />
   <test name="localTerminalApp" type="unit" runInHostApp="true" binary="TerminalApp.LocalTests.dll" />
+  <test name="localSettingsModel" type="unit" runInHostApp="true" binary="SettingsModel.LocalTests.dll" />
+  <test name="unitRemoting" type="unit" runInHostApp="true" binary="Remoting.UnitTests.dll" />

```

Comment:
Wait, what? UnitTest runs in host app? Shouldn't this also be slated to run in Helix then?

Reply:
derp, good catch